dev-requirements.txt

```diff
@@ -5,3 +5,4 @@ requests==2.5.1
 responses==0.3.0
 wheel
 twine
+Django==1.8.5
```

Comment:
We can do that with tox - I actually ripped it out a while back - we should probably tackle re-adding that as a separate task - thoughts?